### PR TITLE
Created unit test

### PR DESCRIPTION
**Pull request by kwmt**
- I created a unit test. so I made some testable changes to flutter_inapp_purchase.dart.
    - ex) added platform package
- I used following sources as a reference.
    - https://github.com/flutter/plugins/blob/master/packages/firebase_storage/test/firebase_storage_test.dart
    - https://github.com/flutter/plugins/blob/master/packages/firebase_messaging/lib/firebase_messaging.dart

- It has nothing to do with this PR, but I think it should be changed to non-static method. The reason is that it is difficult to test. I want to PR the change separately, but  I would like to consult as it will be a breaking change.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -9,6 +9,12 @@ dependencies:
   http: '>=0.12.0 <1.0.0'
   flutter:
     sdk: flutter
+
+  platform: ^2.0.0
```
Comment (hyochan) [inline pubspec.yaml]: Could you guide me on what this is for? I've no idea about this. I hope you can explain this to me..

Comment (kwmt) [reply pubspec.yaml]: Ok,
This plugin is https://pub.dartlang.org/packages/platform. 
It is written in this README, but if you use static properties, when you build on PC(not android or iOS), you can not mock them, so it becomes state that platform not supported.
You can use this plugin to mock it and make it testable.

**Comment by kwmt** [comment]
Thank you for your reviewing and merging. I also think that integrating some ci is a good idea 👍

**Comment by hyochan** [comment]
@kwmt Really appreciate your hard work!

**Pull request merged**